relentless/__init__.py

```diff
@@ -1,7 +1,20 @@
+"""
+Developer tools
+===============
+
+.. automodule:: relentless.collections
+
+.. automodule:: relentless.data
+
+.. automodule:: relentless.math
+
+"""
```

Comment:
This docstring is not correct, as it is supposed to document the whole `relentless` package but instead is documenting a subpart of it. Instead, can you create a page in the docs like `doc/source/api/devtools.rst` and put this content there? Then, you can link that page into the index, at the end of the list ideally.